examples/SendReceiveClient/SendReceiveClient.ino

```diff
@@ -83,7 +135,7 @@ void setup() {
   Serial.println(" mode");
 
   // Set to true if you want debug messages
-  mt_set_debug(false);
+  mt_set_debug(true);
```

Comment:
We'll need to change this function (also in the NodeInfoClient example) to the new macro.